Flux 'interpolate' package (#2148)
influxdata/docs-v2: content/flux/v0.x/stdlib/interpolate/_index.md

---
title: Flux interpolate package
list_title: interpolate package
description: >

>   The Flux `interpolate` package provides functions that insert rows at regular intervals
>   for missing data and interpolate values using different interpolation algorithms.

Reviewer: ```suggestion
  The Flux `interpolate` package provides functions to estimate missing values and insert rows with estimated values in regular intervals.
```
